coordinator/src/tributary/scanner.rs

```diff
@@ -222,7 +222,7 @@ async fn handle_block<D: Db, Pro: Processors>(
                 spec.set().network,
                 CoordinatorMessage::Coordinator(
                   coordinator::CoordinatorMessage::BatchPreprocesses {
-                    id: SignId { key: todo!(), id: data.plan, attempt: data.attempt },
+                    id: SignId { key: todo!(), id: data.plan, block: None, attempt: data.attempt },
```

Comment:
Here you set `block: None`, yet `substrate_signer.rs` sets `block: Some(_)` That means `substrate_signer.rs` and this have different IDs.

Why was the block added to the SignId? I may have already read it yet can't seem to place it exactly.

Reply:
I believe the only reason block hash needed was due to https://github.com/akildemir/serai/blob/0eb4e2a57b287c555e1c52eb376fbe7c4543ce0a/processor/messages/src/lib.rs#L121 this member function on `CoordinatorMessage`